website/docs/general/contributor-guide.md

```diff
@@ -9,45 +9,41 @@ keywords:
 description: This article is a set of guidelines for Apache APISIX contributors, including things that a contributor can do and how to do it well.
 ---
 
-Please fee free to report bugs, submit suggestions, or submit PRs according to this guide.
+Please follow this guide to [report bugs](#submit-an-issue), [submit suggestions](#submit-an-issue), or [submit PRs](#submit-pr).
 
-## Submit an issue
+## Submitting an Issue
 
-1. Before submitting your issues, please go through a comprehensive search to make sure the problem cannot be solved just by searching.
+Before submitting an issue, try solving the problem through a comprehensive search.
 
-2. Check the Issue List to make sure the problem is not repeated.
+[Open issues](https://github.com/apache/apisix/issues) and the [discussions forum](https://github.com/apache/apisix/discussions) are good places to search for solutions before opening an issue.
```

Comment:
Only `discussions` or `discussion forums` will be better choice of words. Not a major issue though. May leave as it is.

Reply:
I used "discussions forum" as GitHub calls it "Discussions" and to make it clear that we mean the forum.